frontend/app/module/data_entry/polling_station/PollingStationLayout.test.tsx

```diff
@@ -50,7 +50,7 @@ describe("PollingStationLayout", () => {
     // Check if the navigation bar displays the correct information
     const nav = await screen.findByRole("navigation", { name: /primary-navigation/i });
     expect(within(nav).getByRole("link", { name: "Overzicht" })).toHaveAttribute("href", "/elections");
-    expect(within(nav).getByRole("link", { name: election.name })).toHaveAttribute(
+    expect(within(nav).getByRole("link", { name: new RegExp(election.name) })).toHaveAttribute(
```

Comment:
```suggestion
    expect(within(nav).getByRole("link", { name: /election.name/ })).toHaveAttribute(
```

RTL [documentation has](https://testing-library.com/docs/queries/about#textmatch) this way of doing regex. We also do it that way in some other tests.

Reply:
never mind, bad idea with variables